### PR TITLE
WIP: Add loaders

### PR DESCRIPTION
**Pull request by irustm**

### CODE REVIEW

File: snippets/loader.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+  };
+
+  constructor(private vcRef: ViewContainerRef, private resolver: ComponentFactoryResolver) { }
+}
+
+```
+
+
+# Bonus
```
Comment (kirjs) [inline snippets/loader.md]: Remove

File: snippets/loader.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  </ng-template>
+
+  Second:
+  <div *ngIf="model">
+    Model
+  </div>
+  <my-loading *ngIf="!model"></my-loading>
+
+  Third:
+  <div *ngIf="model withLoading">
```
Comment (kirjs) [inline snippets/loader.md:35]: Break up into 2 snippets

File: snippets/loader.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  selector: 'my-loading',
+  template: `
+    <div>Loading...</div>
+  `
+})
+export class LoadingComponent { }
+
+
+@Directive({
+  selector: '[ngIfWithLoading]',
```
Comment (thekiba) [inline snippets/loader.md:58]: It's a bad practice when structural directive doesn't take values in Input.

**Comment by kirjs** [comment]
Can we make the content fit into 25 line please?

File: snippets/loader.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Loader options
+
+```typescript
+import { Component, VERSION, Directive, Input, ViewContainerRef, ComponentFactoryResolver } from '@angular/core';
+
+@Component({
+  selector: 'my-app',
+  template: `
+  <button (click)="model = !model">Toggle</button>
```
Comment (kirjs) [inline snippets/loader.md:18]: model -> should this look like a boolean var? e.x. isXxxx

File: snippets/loader.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  - tips
+---
+# Content
+
+Loader options
+
+```typescript
+import { Component, VERSION, Directive, Input, ViewContainerRef, ComponentFactoryResolver } from '@angular/core';
+
+@Component({
```
Comment (kirjs) [inline snippets/loader.md:15]: Can we turn this in a simple structural directive with 1 example? too much stuff otherwise.

**Review by kirjs — CHANGES_REQUESTED** [review]

**Comment by NothingEverHappens** [comment]
I'm trying to recall what was the latest decision on this one. 
I think the plan was to drop it, but feel free to reopen if I am mistaken

**Pull request closed without merging**